examples/edi-order-generator/Generate and Send Outbound EDI Orders.md

```
@@ -0,0 +1,68 @@
+> The `ballerina/edi` 1.6.0 envelope APIs used here are resolved from the local repository
+> (`repository = "local"` in `Ballerina.toml`) until the release is published to Ballerina Central.
+
+## Run the example
+
+```bash
+bal run
+```
+
+Expected output — note `UNT+11` is computed by the module, not supplied by the program:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)